docs/pydoc/config/rankers_api.yml

```
@@ -1,8 +1,7 @@
 loaders:
   - type: haystack_pydoc_tools.loaders.CustomPythonLoader
     search_path: [../../../haystack/components/rankers]
-    modules: ["lost_in_the_middle", "meta_field", "meta_field_grouping_ranker", "transformers_similarity",
-              "sentence_transformers_diversity", "sentence_transformers_similarity"]
+    modules: ["lost_in_the_middle", "meta_field", "meta_field_grouping_ranker", "transformers_similarity", "sentence_transformers_diversity", "hugging_face_tei"]
```

```suggestion
    modules: [ "hugging_face_tei", "lost_in_the_middle", "meta_field", "meta_field_grouping_ranker",  "sentence_transformers_diversity", "sentence_transformers_similarity", "transformers_similarity"]
```

- "sentence_transformers_similarity" should be there
- let's take the opportunity to sort in alphabetical order